perfmetrics/scripts/presubmit_test/pr_perf_test/build.sh

```diff
@@ -15,55 +17,83 @@ set -e
 sudo apt-get update
 echo Installing git
 sudo apt-get install git
-echo Installing python3-pip
-sudo apt-get -y install python3-pip
-echo Installing libraries to run python script
-pip install google-cloud
-pip install google-cloud-vision
-pip install google-api-python-client
-pip install prettytable
 echo Installing go-lang  1.20.5
 wget -O go_tar.tar.gz https://go.dev/dl/go1.20.5.linux-amd64.tar.gz
-sudo rm -rf /usr/local/go && tar -xzf go_tar.tar.gz && sudo mv go /usr/local
+sudo rm -rf /usr/local/go && tar -xzf go_tar.tar.gz && sudo mv go /usr/local &
 export PATH=$PATH:/usr/local/go/bin
-echo Installing fio
-sudo apt-get install fio -y
 
 # Run on master branch
```

Comment:
This comment needs to go. It is misplaced after the master branch run section has been moved down.